### ISSUE TITLE
Routine update of dependencies

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2875]
Update Poetry managed dependencies within existing pyproject.toml constraints.

```
cd /opt/rockstor
poetry update
```

Issue resolution via resulting updated poetry.lock file.

---

Upstream reference: https://python-poetry.org/docs/basic-usage/#updating-dependencies-to-their-latest-versions

**Comment by phillxnet** [comment]
# Resulting Poetry report
Leap 15.6 X86_64 host

```
Package operations: 0 installs, 1 update, 0 removals

  • Updating sqlparse (0.5.0 -> 0.5.1)

Writing lock file
```

**Comment by phillxnet** [comment]
# Notable Changelog references:

 - https://sqlparse.readthedocs.io/en/latest/changes.html

**Comment by phillxnet** [comment]
Closing as:
Fixed by #2876
